standalone/job_status_detail.py

```diff
@@ -1,136 +1,161 @@
-	pig_params['aps'] = cfg_path + args.tenant + '_' + job_set[0] + '_ap.json';
-	pig_params['dt'] = args.date 
-	pig_params['mode'] = mode
-	pig_params['n_eg'] = ArConfig.get('datastore_mapping','n_eg')
-	pig_params['n_gg'] = ArConfig.get('datastore_mapping','n_gg')
-	pig_params['n_alt'] = ArConfig.get('datastore_mapping','n_alt')
-	pig_params['n_altf'] = ArConfig.get('datastore_mapping','n_altf') 
-	pig_params['sd_map'] = ArConfig.get('datastore_mapping','sd_map')
-	pig_params['flt'] = '0'
-	pig_params['mongo_status_detail'] = 'mongodb://' + mongo_host + ':' + mongo_port + '/' + mongo_dest;
-	
-	cmd_pig = []
-
-	# Append pig command
-	cmd_pig.append('pig')
-
-	# Append Pig local execution mode flag
-	if ar_mode == "local":
-		cmd_pig.append('-x')
-		cmd_pig.append('local')
-
-	# Append Pig Parameters
-	for item in pig_params:
-		cmd_pig.append('-param')
-		cmd_pig.append(item+'='+pig_params[item])
-
-	# Append Pig Executionable Script
-	cmd_pig.append('-f')
-	cmd_pig.append(pig_script_path+'compute-status.pig')
-
-	# Command to clean a/r data from mongo
-	cmd_clean_mongo_status = [os.path.join(stdl_exec,"mongo_clean_status.py"),'-d',args.date]
-
-	# Command to upload sync data to hdfs
-	cmd_upload_sync = [os.path.join(stdl_exec,"upload_sync.py"),'-d',args.date,'-t',args.tenant,'-j',job_set[0]]
-
-	# Command to clean hdfs data
-	cmd_clean_sync = ['hadoop','fs','-rm','-r',hdfs_sync_path]
-
-	# Upload data to hdfs
-	print "Uploading sync data to hdfs..."
-	call(cmd_upload_sync)
-
-	# Clean data from mongo
-	print "Cleaning data from mongodb"
-	call(cmd_clean_mongo_status)
-
-	# Call pig
-	print "Submitting pig compute status detail job..."
-	call(cmd_pig)
-
-	# Cleaning hdfs sync data 
-	if sync_clean == "true":
-		print "System configured to clean sync hdfs data after job"
-		call(cmd_clean_sync)
-
-	print "Excution of status job for tenant %s for date %s completed!" % ( args.tenant, args.date)
+    # default config
+    fn_ar_cfg = "/etc/ar-compute-engine.conf"
+    arsync_exec = "/usr/libexec/ar-sync/"
+    arsync_lib = "/var/lib/ar-sync/"
+    arcomp_conf = "/etc/ar-compute/"
+    arcomp_exec = "/usr/libexec/ar-compute/"
+    stdl_exec = "/usr/libexec/ar-compute/standalone"
+    pig_script_path = "/usr/libexec/ar-compute/pig/"
+
+    actual_date = datetime.strptime(args.date, '%Y-%m-%d')
+    one_day_ago = actual_date - timedelta(days=1)
+    prev_date = one_day_ago.strftime('%Y-%m-%d')
+    prev_date_under = prev_date.replace("-", "_")
+
+    date_under = args.date.replace("-", "_")
+
+    ArConfig = SafeConfigParser()
+    ArConfig.read(fn_ar_cfg)
+
+    # Initialize logging
+    log_mode = ArConfig.get('logging', 'log_mode')
+    log_file = ArConfig.get('logging', 'log_file')
+    log_level = ArConfig.get('logging', 'log_level')
+    logger = init_logger(log_mode, log_file, log_level, '[job_cycle.py]')
+
+    # Initialize logging
+    log_mode = ArConfig.get('logging', 'log_mode')
```

Comment:
lines 41-43 seem duplicates of lines 35-37. Maybe it's ok but please check if the second part is not needed.